packages/web/src/components/auth/Login.tsx

```diff
@@ -84,7 +89,7 @@ const Login = () => {
                 color='primary'
                 mt='10px'
                 type='submit'
-                onClick={handleSubmit}
+                onClick={() => handleSubmit()}
```

Comment:
is this needed?

Reply:
I changed to fix the following typescript error:
![Screenshot_20210222_084836](https://user-images.githubusercontent.com/48305450/108704512-e2224c80-74ea-11eb-951c-e678d8087402.png)

Reply:
nice trick